sky/tools/create_macos_gen_snapshots.py

```diff
@@ -16,6 +16,7 @@ def main():
   )
 
   parser.add_argument('--dst', type=str, required=True)
+  parser.add_argument('--clang-dir', type=str, required=True)
```

Comment:
The bots call this script, so to make this a soft transition, the new flag would need to be option with a default value of 'clang_x64'.

Reply:
Yay! That explains the mystery errors that had no logs.

Reply:
But this begs the question - if/when we start having Apple Silicon machines running our CI, won't this just shift the problem from needing a default to needing the create_macos_gen_snapshots.py script be able to figure out the default?

Reply:
Or teaching the CI machines to figure this out and provide the answer when they call the script? Maybe that's the best strategy?

Reply:
Yeah, we'll have to modify the recipe to get it to work on arm64.

Reply:
Is it worth updating the recipes now?

Reply:
After this lands, we should update to explicitly pass the new flag, but I think we should wait to add logic to programatically figure out the arch until we're ready to bring those machines online.